src/core/libmaven/Scan.cpp

```diff
@@ -15,6 +15,7 @@ Scan::Scan(mzSample* sample, int scannum, int mslevel, float rt, float precursor
 	this->precursorCharge=0;
 	this->precursorIntensity=0;
     this->sampleNumber = -1;
+    this->originalRt = -1;
```

Comment:
@IpankajI Better to assign the originalRt here instead of doing this in mzAligner.

Reply:
@shubhra-agrawal done!